seirmo/apps/seir_simulation_app.py

```diff
@@ -68,26 +69,34 @@
  infecting rate of the disease, many models are used to characterise the
  disease, such as the SEIR model and the branching processes model.  
   
-SEIR model is a deterministic model and has been used to model infectious
- disease.  """
+SEIR model is a deterministic model. A deterministic model means that the simulation will always 
+be the same as long as the parameters are the same. This SEIR model has been used to model 
```

Comment:
"This SEIR model has been used to model infectious disease." can go I think, because you mention that above already.